### PR TITLE
[hi] Added content/hi/docs/concepts/architecture/_index.md

### PR DESCRIPTION
**Pull request by akash-kumar-saw**
Issue Number #39548

Localized en/docs/concepts/architecture/_index.md in hindi language :)

### CODE REVIEW

File: content/hi/docs/concepts/architecture/_index.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+title: क्लस्टर आर्किटेक्चर
+weight: 30
+description: >
+  कुबेरनेट्स के पीछे वास्तु अवधारणाएँ।
+---
+
```
Comment (dipesh-rawat) [inline content/hi/docs/concepts/architecture/_index.md:7]: It seems that the localization of this file may be incomplete. There is a diagram included in the `{{ <figure> }} ` shortcode in the English file _[(check it out on Line 8)](https://github.com/kubernetes/website/blob/17f080049278a691d417c44decddbdb297f744b5/content/en/docs/concepts/architecture/_index.md?plain=1)_ that should be incorporated into this Hindi file here.

File: content/hi/docs/concepts/architecture/_index.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+{{< figure src="/images/docs/kubernetes-cluster-architecture.svg" alt="Components of Kubernetes" caption="Kubernetes cluster architecture" class="diagram-large" >}}
```
Comment (utkarsh-singh1) [inline content/hi/docs/concepts/architecture/_index.md:8]: Hi @akash-kumar-saw, can you localize caption into Hindi.

Comment (utkarsh-singh1) [reply content/hi/docs/concepts/architecture/_index.md:8]:
```suggestion
{{< figure src="/images/docs/kubernetes-cluster-architecture.svg" alt="Components of Kubernetes" caption="कुबेरनेट्स क्लस्टर आर्किटेक्चर" class="diagram-large" >}}
```

**Review by divya-mohan0209 — CHANGES_REQUESTED** [review]
Hello @akash-kumar-saw : Please advise if you have the bandwidth to localise the caption into Hindi?

**Comment by divya-mohan0209** [comment]
@akash-kumar-saw Since there has been no response/activity on this PR, we'll be closing it. If you have the bandwidth to continue work on this PR, please feel free to reopen the PR and let us know.
/close

**Pull request closed without merging**